scrapy/extensions/memusage.py

```diff
@@ -92,6 +93,8 @@ def _check_limit(self):
                 self.crawler.engine.close_spider(self.crawler.engine.spider, 'memusage_exceeded')
             else:
                 self.crawler.stop()
+        else:
+            logger.info("Current memory usage is %(virtualsize)dM", {'virtualsize': current_mem_usage / 1024 / 1024})
```

Comment:
I think this message might be misleading. We're reporting ru_maxrss, which is the max memory used by the current process over its lifetime, not the current memory usage.